migrate/20250115_project_id_name_indexes.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  no_transaction
+
+  up do
+    %i[postgres_resource firewall private_subnet vm].each do |table|
```

Comment:
Why isn't `minio_cluster` included in this list?

Reply:
No good reason.  I'll add it.